### DISCUSSION TITLE
Database v1 tables have not been detected.

### DISCUSSION THREAD

**Discussion by nkilm** [umami-software/umami#1920] [General]
Hey I've followed all the instructions mentioned in [migration guide](https://umami.is/docs/migrate-v1-v2), but I still get the following error. 

![image](https://user-images.githubusercontent.com/79012023/233166576-b26fe611-4b31-4ce7-91ba-649b1638c16a.png)

I have all the tables which were created in by umami v1. See the below screenshot displaying all the tables. 

![image](https://user-images.githubusercontent.com/79012023/233167293-7c60fcdc-58b4-4731-bfa3-c6e2f2d78a30.png)

I am using [planetscale](https://planetscale.com) to store all umami data. v1 is working smoothly with no errors. Also I am deployed the site to [vercel](https://vercel.com).

**Comment by franciscao633** [comment]
planetscale is tricky because of the foreign key constraints did not mix well with prisma migrations. Can you check your _prisma_migrations table and let me know what records are there.

**Comment by nkilm** [comment]
I don't have `_prisma_migrations` table in my database😅

![image](https://user-images.githubusercontent.com/79012023/233254316-20e5eae5-87f8-4ce2-a46b-934ac1d0c446.png)

I followed this guide https://umami.is/docs/running-on-vercel and deployed directly on vercel.

**Comment by nkilm** [comment]
How do I update my schema to v1(latest) for my planetscale db? It is mentioned in the [docs](https://umami.is/docs/migrate-v1-v2#:~:text=Database%20schema%20must%20be%20in%20sync%20with%20the%20latest%20v1%20Umami%20repo.). I am not sure how to check if the schema in my database is of latest version for v1 or not.

**Comment by franciscao633** [comment]
Can you check if you event / account table have UUID columns? If so I think you should have the latest and you can follow the steps below. I'm hoping we can get planetscale to work.

1. Inside your Umami folder run `yarn build-db`
2. then run `npx prisma migrate resolve --applied 04_add_uuid`
3. the run the migration `npx @umami/migrate-v1-v2@latest`

**Reply by nkilm** [reply]
Yeah both event and account tables have `uuid` column.

**Reply by nkilm** [reply]
But the command you provided is not working. 

![image](https://user-images.githubusercontent.com/79012023/233422584-82b47709-1337-414a-b749-9bf8d6bdd166.png)

I don't have that migration.

![image](https://user-images.githubusercontent.com/79012023/233422761-8b6db0d1-4a54-40b6-959e-c3469d55de48.png)

**Comment by franciscao633** [comment]
I apologize. The 04_add_uuid migration only exists in v1.4. You would need to run all those commands on v1.4, then upgrade your app to 2.0.

**Reply by nkilm** [reply]
Where do I find those commands?

**Reply by franciscao633** [reply]
Some are prisma commands, which can be found [here](https://www.prisma.io/docs/guides/migrate/production-troubleshooting). Also `package.json` contains commands under scripts.

**Comment by nkilm** [comment]
@franciscao633 how do I check which version of the database schema do I have? In which table is this stored?
